main.js

```diff
@@ -1,45 +1,100 @@
-$(document).ready(function(){
-    $('.phone-number').mask('+7(000) 000-00-00');
-    });
+// testing
+    $(document).ready(function(){
+        $('.phone-number').mask('+7(000) 000-00-00');
+        });
+    
 
- let emailValid = /^([a-z0-9_\.-])+@[a-z0-9-]+\.([a-z]{2,4}\.)?[a-z]{2,4}$/;
-		$('#exampleInputEmail').keyup(function(){
-            !emailValid.test($('#exampleInputEmail').val())
-            ? $('#exampleInputEmail').css('border-color' , 'rgb(255, 0, 0)')
-            : $('#exampleInputEmail').css('border-color', '#007d7c');
-		});
+    let inputEmail = document.getElementById('inputEmail');
+    let pass = document.getElementById('inputPassword');
+    let passConfirm = document.getElementById('inputPasswordConfirm');
+    let showHidden = document.getElementById('showHidden');
+    let showHiddenConfirm = document.getElementById('showHiddenConfirm');
+    let inputNameValue, inputEmailValue, inputPhoneValue, inputPasswordValue;
+    let result = document.getElementById('result');
+    let formContent = document.getElementById('formContent');
+    
+    let setListener = (element, type, hendler) => {
+        if (!element){
+        return;
+    }
+    element.addEventListener(type, hendler);
+    return () => {
+        element.removeEventListener(type, hendler);
 
-let pass = document.getElementById('exampleInputPassword');
-let passConfirm = document.getElementById('exampleInputPasswordConfirm');
+    }
+}
+ 
+    function check() {
+        if (pass.value !== passConfirm.value) {
+            (pass, passConfirm).style.color = "red"
+        } else {
+            (pass, passConfirm).style.color = "black"
+        }
+    };
+    
+    function showPassword(){
+           
+        if (pass.getAttribute('type') == 'password') {
+            showHidden.classList.add('view');
+            pass.setAttribute('type', 'text');
+        } else {
+            showHidden.classList.remove('view');
+            pass.setAttribute('type', 'password');
+        }
+        return false;
+    };
+    
+    function showPasswordConfirm(){
+           
+        if (passConfirm.getAttribute('type') == 'password') {
+            showHiddenConfirm.classList.add('view');
+            passConfirm.setAttribute('type', 'text');
+        } else {
+            showHiddenConfirm.classList.remove('view');
+            passConfirm.setAttribute('type', 'password');
+        }
+        return false;
+    };
 
-passConfirm.onkeyup = function check() {
-    if (pass.value !== passConfirm.value) {
-        (pass, passConfirm).style.color = "red"
-    } else {
-        (pass, passConfirm).style.color = "black"
+    setListener(passConfirm,"keyup", check);
```

Comment:
fix indentation, use single quotes for strings for all project, missed whitespace